### ISSUE TITLE
Feature-Request: Remove Docker dependency

### ISSUE THREAD

**Issue by Nold360** [soxfor/qbittorrent-natmap#14]
Hi,

i would like to use your container on my kubernetes cluster. So i don't have docker.sock available.

I might start hacking the start.sh script so it works without the need of docker. But just in case i wanted to create this issue/FR

**Comment by FunDeckHermit** [comment]
I'm running qBittorrent in a VM, so no need for containers. Tried stripping docker but no luck.

**Comment by bertho-zero** [comment]
I vote for it too, it's never really good to depend on docker.sock

**Comment by naimo** [comment]
Agreed, mounting docker.sock [is bad practice ](https://news.ycombinator.com/item?id=17983623)

Looking at the script, it seems to need it to
- check if the vpn container is healthy
=> could use the glutun endpoint instead on port 9999 https://github.com/qdm12/gluetun-wiki/blob/main/faq/healthcheck.md
- modify iptable rules inside the vpn container
=> I don't know enough about gluetun's firewall settings but it seems weird to be adding firewall rules on ports when ProtonVPN already only opens the desired port
